support-workers/package.json

```diff
@@ -9,10 +9,11 @@
     "lint": "eslint src/typescript/**/*.ts",
     "check-formatting": "prettier --check **.ts",
     "build": "tsc",
+    "bundle": "esbuild src/typescript/lambdas/*.ts --bundle --platform=node --target=node22 --outdir=target/typescript",
```

Comment:
I have introduced esbuild as a bundler because it was very difficult to get pnpm to produce a flat set of dependencies for deployment. It should also produce a smaller deployment artifact.